README.md

```diff
@@ -14,14 +14,34 @@
 
 ## Live Demo
 
-[Live Demo Link]()
+[Live Demo Link](https://nyame-wolf.github.io/Leaderboard/)
```

Comment:
- Please, kindly ensure the live demo link takes the user to the live version of your leaderboard app. You can choose to update the link or remove it to improve the user experience and to make your readme more professional.

Reply:
- Please, kindly add the "Acknowledgement" section and add it's contents to make your readme more professional.